README.md

```diff
@@ -8,4 +8,17 @@
   
   2.And also  setup the [virtualenv](https://docs.python.org/3/tutorial/venv.html) in your local directory.
 
-  3.And finally run the application with command as "python3 -m flask run"
\ No newline at end of file
+  3.And finally run the application with command as "python3 -m flask run"
+
+  ## How to Run:
+   1.Once a collaborator pulls down your project - they can then install a fresh python environment that is,
+
+      "python3 -m venv local_python_environment"
```

Comment:
You can use simpler env name like `venv` instead of ocal_python_environment

Reply:
Yes, we use local_python_environment instead of 'venv'

Reply:
sorry.., we use **venv** instead of local_python_environment